lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -238,6 +238,11 @@ def __init__(self, *args, distro: Type[OpenSearchDistribution] = None):
         self.framework.observe(self.on.set_password_action, self._on_set_password_action)
         self.framework.observe(self.on.get_password_action, self._on_get_password_action)
 
+        # There is an explosion of peer_relation_changed calls at the startup.
+        # As this corresponds to a local peer relation (not the peer-cluster), we can safely
+        # abandon repeated events.
+        self._local_peer_relation_changed_has_deferred = False
```

Comment:
just for the sake of conciseness 
```suggestion
        self._is_peer_rel_changed_deferred = False
```